app/views/categories/index.html.erb

```diff
@@ -15,6 +14,19 @@
 	<div class="row">
 		<div class="col-md-2 center-block"></div>
 			<% @categories.each do |category| %>
+			<% if category.words.count == 0 %>
+				<div class="col-md-3 center-block">
+					<div id="lesson-box-prep">
+						<h4>Title</h4>
+			   			<h4><strong><%= category.title %></strong></h4><hr>
+			   			<h4 class="mt-5">Description</h4>
+				    	<h5 style="height: 200px;"><%= category.description %></h5>
+				    </div>
+				    <div class="text-center mt-4 mb-1">
+				    <h3><strong>Getting Ready</strong></h3>
+				    </div>	    	
+				</div>
+			<% else %>
 				<div class="col-md-3 center-block">
 					<div id="lesson-box">
```

Comment:
@nozomi-matsuoka 
You can create a `ternary expression`, to avoid redundancy
`<%= category.words.any? ? "lesson-box" : "lesson-box-prep" %>`

And also, change the `id` to `class`